main.js

```diff
@@ -34,16 +34,29 @@ function extractId(url) {
  * @param {import("puppeteer").Page} page
  * @param {string} selector
  * @param {boolean} [isTable]
+ * @param {boolean} [isPlayerInfo]
  */
-async function extractHeader(page, selector, isTable) {
+async function extractHeader(page, selector, isTable, isPlayerInfo) {
     const result = {};
-    const rows = await page.$$(selector + (isTable ? ' tr' : ' p'));
-    for (const row of rows) {
-        const cells = await row.$$(isTable ? 'th, td' : 'span');
-        if (cells.length > 1) {
-            const name = convertName(await getText(cells[0]));
-            const rText = (await getText(cells[1])).split(/\s\s/);
-            result[name.slice(0, name.length - 1)] = rText.length > 1 ? rText : rText[0];
+    if (isPlayerInfo) {
+        const keyRows = await page.$$(`${selector}--regular`);
+        const valRows = await page.$$(`${selector}--bold`);
+
+        for (const [index, item] of keyRows.entries()) {
+            const keyItem = convertName(await getText(item));
+            const valItem = (await getText(valRows[index])).split(/\s\s/);
+            result[keyItem.slice(0, keyItem.length - 1)] = valItem.length > 1 ? valItem : valItem[0];
```

Comment:
This is a bit too complex, you can simplify by putting `keyItem.slice(0, keyItem.length - 1)` to separate line